sdk/core/System.Net.ClientModel/tests/clients/System.Net.ClientModel.Tests.Clients.csproj

```diff
@@ -1,19 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
+		<Description>Test client code for use in System.Net.ClientModel test libraries.</Description>
```

Comment:
This is where we can add any code intended to simulate what would live in a client project, i.e. test models for testing model reader writer interfaces.  It doesn't have nullability enabled and it doesn't have access to library internals, so it will be a higher fidelity simulation than what's possible in our other libraries.  Both the public and internal test projects take a dependency on this library.